### ISSUE TITLE
[BUG] Output all 'Export' fields failed to proceed

### ISSUE THREAD

**Issue by trumpy81** [WebTools-NG/WebTools-NG#643]
**Describe the bug**
I added all 'Export" fields to a custom level for TV Episodes and output the range of 0 > 1000. At item 383, the output stalled and did not proceed beyond that point. All artwork was output up to that point.

**To Reproduce**
1. Select Custom Levels > TV Episodes > define new level
2. Add all available 'Export' fields + Series Title + Title + Episode
3. Export using this Custom Level
4. Set range from 0 > 1000
5. See error

**Expected behavior**
I would expect the output to proceed and not stall.

**Screenshots**
![Screenshot 2022-12-02 013835](https://user-images.githubusercontent.com/12381786/205096838-0878301f-a14d-4871-8956-cd60d449cf55.png)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro

**Logs**
[logs.zip](https://github.com/WebTools-NG/WebTools-NG/files/10133861/logs.zip)

**Note:**
If it helps, it stalled at Air Australia - Air Australia - 01 - Canvas & Sticks.mp4.

I tried moving the 3 files into a Season 01 folder and renaming the files in question, but that made no difference, it still stalled at item 383.

I also did a Fix Match and Refresh Metadata, but again, it stalled at item 383,

**Comment by ukdtom** [comment]
Lost access to your server?

**Comment by trumpy81** [comment]
My network settings on the PC went berserk and it left Plex without a connection or Remote Access.

Hopefully I have restored it now, but let me know if you still cannot connect.

**Comment by ukdtom** [comment]
You're back in business'

**Comment by trumpy81** [comment]
Phew, thank the powers that be for that ... lol :)

**Comment by ukdtom** [comment]
Hi Andy.

I did an export from your box, from item 382 to to 390

And yes, your `Air Australia` is indeed missing some posters and arts, but it didn't make my export stop!

Attached, you can see the export, and also see, that all places where I did fail, the jpg file was replaced with a txt file to indicate this

What is strange though, is that your log doesn't indicate the error that we need to do the replace...

So strange, and will look into this asap

Please don't change your serie, since the only place we can reproduce this

[ROGStrix.zip](https://github.com/WebTools-NG/WebTools-NG/files/10136870/ROGStrix.zip)

**Comment by trumpy81** [comment]
I have switched Air Australia to The Movie Database (aired) ordering. I also uploaded artwork to the TMDB listing for Air Australia. That will take a few days to filter down to Plex, but after that, fingers crossed, there should not be any missing artwork.

The TVDB only has the poster for Air Australia, nothing else.

I need to reboot the PC shortly, when I return I will perform the output again and let you know the results. I did expect to see the .txt files as well, but sadly, they did not show up.
